config/settings.yml

```diff
@@ -17,7 +17,7 @@ argocd:
     api_key: <%= ENV['argocd__slack__api_key'] %>
 ask_va_api:
   crm_api:
-    auth_url: https://login.microsoftonline.us
+    auth_url: <%= ENV['ask_va_api__crm_api__auth_url'] %>
```

Comment:
I see this in AWS for all envs except Sandbox. Can you add the Sandbox variable as well?